helm/cluster-vsphere/templates/kubevip-staticpod-template.yaml

```diff
@@ -24,7 +24,7 @@ stringData:
         - name: vip_address
           value: {{ .Values.connectivity.network.controlPlaneEndpoint.host }}
         - name: vip_interface
-          value: eth0
+          value: ens192
```

Comment:
we would have to change it for CPI uber-chart here - https://github.com/giantswarm/cloud-provider-vsphere-app/blob/main/helm/cloud-provider-vsphere/values.yaml#L24

Reply:
Yeah. Just noticed it. Will take care.

Reply:
I set it here https://github.com/giantswarm/cluster-vsphere/pull/98/commits/c38ba6f3b13683d0a10913d11f60985ba75b7b56 

After switching Flatcar fully, I will change the default on CPI side too.